data_model.md

```diff
@@ -288,30 +304,33 @@ fields determined by the data’s type listed below:
 #### Observations
 {: .anchor}
 
--   *Timestamp* - A timestamp in ISO-8601 format including a timezone. e.g. `2018-12-05T23:01:45-07:00` or `20181206T060145Z`.
+-   *Timestamp* - A timestamp in ISO-8601 format including a timezone.
+    e.g. `2018-12-05T23:01:45-07:00` or `20181206T060145Z`.
 -   *Value* - Values for the variable in units defined above.
--   *Quality Flag* - A flag indicating if the value is questionable. Uploads may contain values 0 (ok) or 1 (questionable). Downloads may contain additional flags determined by the data qualification toolkit (discussed elsewhere).
+-   *Quality Flag* - A flag indicating if the value is questionable. Uploads
+    may contain values 0 (ok) or 1 (questionable). Downloads may contain
+    additional flags determined by the data qualification toolkit (discussed
+    elsewhere).
 
 #### Forecasts
 {: .anchor}
 
 Single-valued forecasts (e.g. mean or 50<sup>th</sup> percentile) comprise a
 series of Timestamp, Value pairs in the following format:
 
--   *Timestamp* - A timestamp in ISO-8601 format including a timezone. e.g. `2018-12-05T23:01:45-07:00` or `20181206T060145Z`.
+-   *Timestamp* - A timestamp in ISO-8601 format including a timezone.
+    e.g. `2018-12-05T23:01:45-07:00` or `20181206T060145Z`.
 -   *Value* - Values for the variable in units defined above.
 
-Probabilistic forecasts comprise the same set of Timestamp, Value pairs
-as deterministic forecasts *for each probability bin*. To upload a
-complete probabilistic forecast, users upload one forecast per
-probability bin. The metadata of the probabilistic forecast defines the
-*distribution type* (cumulative distribution function (CDF) or
-probability distribution function (PDF) and the *bins*. For a CDF, the
-Value is the quantity of MW, GHI, etc. for the quantile associated with
-each individual forecast upload. For a PDF, the Value is the probability
-corresponding the interval range of MW, GHI, etc. for each individual
-forecast upload. The full set of metadata required to describe
-probabilistic forecasts will be determined later.
+##### Probabilistic forecasts
+{: .anchor}
+
+To upload a complete probabilistic forecast, users upload one
+[forecast](#forecasts-1) for each of the probalistic forecast's [*constant
+values*](#probabilistic-forecasts). The format of each upload is the
+same, but the upload destination specified by the Solar Forecast Arbiter
+is different.
```

Comment:
Why is the upload destination different? I would think that it would still be /forecasts with the appropriate id

Reply:
isn't it /forecasts/9812374878723180974890172 vs /forecasts/66t3uy4tr7178236479826?

Reply:
Yes, now I understand what you mean